internal/server/routes.go

```diff
@@ -215,12 +216,20 @@ func wrapRoute[Req, Res any](a *API, routeID routeIdentifier, route route[Req, R
 			return err
 		}
 
+		org := getRequestContext(c).Authenticated.Organization
 		if !route.noOrgRequired {
-			if org := getRequestContext(c).Authenticated.Organization; org == nil {
+			if org == nil {
 				return internal.ErrBadRequest
 			}
 		}
 
+		if org != nil {
+			// TODO: limit should be a per-organization setting
+			if err := redis.RateOK(a.server.redis, org.ID.String(), 3000); err != nil {
```

Comment:
Is this TODO still relevant?

Reply:
Yes it's still relevant. The rate limit may eventually be different per org so this will need to load the value from settings